content/docs/start/experiments.md

```diff
@@ -43,71 +42,87 @@ evaluate:
-value to the `scores.json` file which is marked as a metrics file with `-M`:
+[`evaluate.py`](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py)
+writes the model's
+[AUC value](https://towardsdatascience.com/understanding-auc-roc-curve-68b2303cc9c5)
+to `scores.json`, which is marked as a metrics file with `-M`:
 
 ```json
 { "auc": 0.57313829 }
 ```
 
-And it
-[writes](https://github.com/iterative/example-get-started/blob/master/src/evaluate.py#L31)
-`precision`, `recall`, `thresholds` arrays it gets with the
-[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html)
-call into the `prc.json`:
+It also writes `precision`, `recall`, and `thresholds` arrays (obtained using
+[`precision_recall_curve`](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.precision_recall_curve.html))
+into plots file `prc.json`:
 
 ```json
 {
   "prc": [
     { "precision": 0.021473008227975116, "recall": 1.0, "threshold": 0.0 },
-    ...{ "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
+    ...
+    { "precision": 1.0, "recall": 0.009345794392523364, "threshold": 0.64 }
   ]
 }
 ```
 
 > DVC doesn't force you to use any specific file names, or even format or
-> structure of a metrics or plots file - it's pretty much user and case defined,
-> please, refer to `dvc metrics` and `dvc plots` documentation for more details.
+> structure of a metrics or plots file - it's pretty much user and case defined.
+> Please refer to `dvc metrics` and `dvc plots` for more details.
 
-Let's save this experiment to compare it later using DVC comparison commands:
+Let's save this experiment, so we can compare it later:
 
 ```dvc
 $ git add scores.json prc.json
 $ git commit -a -m "Create evaluation stage"
 ```
 
-We've managed to capture metrics that experiment has, and later we will see how
-this information can be used to compare different experiment iterations,
-visualize them (e.g. `ROC` or model loss plots). Let's see how can we capture
-another important piece of information that will be important to compare
-experiments - experiment parameters.
+Later we we will see how these and other can be used to compare and visualize
+different experiment iterations. For now, let's see how can we capture another
+important piece of information that will be useful to compare experiments:
+parameters.
 
 ## Defining parameters
 
-It's pretty common for data processing pipelines to use a separate YAML or JSON
-config file that defines adjustable parameters you use to train a model, do
-pre-processing, etc. DVC provides a mechanism for a stage to depend on values
-from such a config file. That's how stage is defined in the `dvc.yaml` file:
+It's pretty common for data science pipelines to include configuration files
+that defines adjustable parameters to train a model, do pre-processing, etc. DVC
```

Comment:
you making everything a bit too formal to my taste (here, in some other places) :)

like config file vs configuration files.

anyways, `defines` -> `define` in your case

Reply:
I do have a weakness for using complete words once at least, I'm OK with abbreviations later. Probably I shouldn't force this sort of thing on other people's text, sorry. It's just a habit at this point.

And yes, changing to "define". Thanks